compose/env-file.md

```diff
@@ -5,21 +5,25 @@ title: Declare default environment variables in file
 notoc: true
 ---
 
-Compose supports declaring default environment variables in an environment
-file named `.env` placed in the folder `docker-compose` command is executed from
+Compose supports declaring default environment variables in an environment file
+named `.env` placed in the folder where the `docker-compose` command is executed
 *(current working directory)*.
 
-Compose expects each line in an env file to be in `VAR=VAL` format. Lines
-beginning with `#` (i.e. comments) are ignored, as are blank lines.
+### Syntax Rules
 
-> Note: Values present in the environment at runtime will always override
-> those defined inside the `.env` file. Similarly, values passed via
-> command-line arguments take precedence as well.
+These syntax rules apply to the `.env` file:
+
+* Compose expects each line in an `env` file to be in `VAR=VAL` format.
+* Lines beginning with `#` (i.e. comments) are ignored.
+* Blank lines are ignored.
+* There is no special handling of quotation marks.
+
+### Compose file and CLI variables
 
-Those environment variables will be used for
-[variable substitution](compose-file.md#variable-substitution) in your Compose
-file, but can also be used to define the following
-[CLI variables](reference/envvars.md):
+The environment variables you define here will be used for [variable
+substitution](compose-file/index.md#variable-substitution) in your Compose file,
+but can also be used to define the following [CLI
```

Comment:
I know this was in the previous version, but I think "but" here can be "and" instead (the 2 features are not mutually exclusive)